server/templates/staff/course/students.list.html

```diff
@@ -37,6 +38,11 @@ <h3 class="box-title"><span> {{ title }} </span></h3>
                     <td class="role"><span class="label label-success">{{item.role}}</span></td>
                     <td class="timestamp" data-timestamp="{{ item.created }}">{{ utils.local_time(item.created, current_course) }}</td>
                     <td class="section">{{item.section}}</td>
+                    <td class="unenroll">
+                        {% call forms.render_form(unenroll_form, action_url=url_for('.unenrollment', cid=current_course.id, user_id=item.user.id), action_text='Unenroll',
```

Comment:
Should we have a confirm dialog too? Seems like it's easy to accidentally delete a student